models/models.js

```diff
@@ -33,6 +33,22 @@ exports.fetchUsers = () => {
   });
 };
 
+exports.fetchAllArticles = (topic) => {
+  let queryString = `SELECT articles.*, COUNT(comment_id)::INT AS comment_count
```

Comment:
Don't send body back on the articles endpoint -> I know this isn't mentioned in the trello ticket but we purposely don't want to send all of that text back on this big GET request so try to be specific with your select